releases/fluentd-splunk-objects.yaml

```
@@ -0,0 +1,54 @@
+releases:
+
+#######################################################################################
+## Fluentd to Splunk logs                                                            ##
+## Forward logs to Splunk with fluentd                                               ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/splunk/splunk-connect-for-kubernetes/blob/master/helm-chart/splunk-kubernetes-logging/values.yaml
+#
+
+- name: "fluentd-splunk-objects"
+  namespace: "kube-system"
+  labels:
+    chart: "splunk-kubernetes-objects"
+    repo: "splunk"
+    component: "splunk"
+    namespace: "kube-system"
+    default: "false"
+  chart: "https://github.com/splunk/splunk-connect-for-kubernetes/releases/download/1.1.0/splunk-kubernetes-objects-1.1.0.tgz"
+  wait: true
+  installed: {{ env "FLUENTD_SPLUNK_INSTALLED" | default "true" }}
+  values:
+  - fullnameOverride: "splunk-k8s-objects"
```

```suggestion
  - fullnameOverride: "fluentd-splunk-objects"
```

I think it's more confusing to introduce yet another variation that is not any shorter and is not needed for deterministic service discovery

---

I updated the names so that the services etc. have the names Splunk support would be expecting.